recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+    "    return ('COLUMN_PHASE: ' + str(phase_column_name) \n",
+    "+ '; PINT_VERS: ' + pint.__version__ \n",
+    "+ '; GAMMAPY_VERS: ' + gammapy.__version__ \n",
+    "+ '; EPHEM_FILE: ' + ephemeris_file \n",
+    "+ '; PSRJ :' + str(model.PSR.value) \n",
+    "+ '; START: ' + str(model.START.value) \n",
+    "+ '; FINISH: ' + str(model.FINISH.value) \n",
+    "+ '; TZRMJD: ' + str(model.TZRMJD.value) \n",
+    "+ '; TZRSITE: ' + model.TZRSITE.value \n",
+    "+ '; TZRFREQ: ' + str(model.TZRFRQ.value) \n",
+    "+ '; EPHEM: ' + str(model.EPHEM.value) \n",
+    "+ '; RA: ' + str(model.RAJ.value) \n",
+    "+ '; DEC: ' + str(model.DECJ.value) \n",
+    "+ '; DATE: ' + str(Time.now().mjd) \n",
+    "+ '; PHASE_OFFSET: ' + 'default = 0' + '; ')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "8c866a5c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "COLUMN_PHASE: PHASE; PINT_VERS: 0.9.3; GAMMAPY_VERS: 1.0; EPHEM_FILE: ./0534+2200_ApJ_708_1254_2010.par; PSRJ :J0534+2200; START: 54686.1526259; FINISH: 56583.1591704; TZRMJD: 55638.155277599951656; TZRSITE: coe; TZRFREQ: inf; DMDATA: False; INSTRUMENT: TODO; EPHEM: DE405; RA: 5.575538888888889; DEC: 22.01447222222222; DATE: 59998.37212295684; PHASE_OFFSET: default = 0; \n"
+     ]
+    }
+   ],
+   "source": [
+    "phase_log = get_log(ephemeris_file=ephemeris_file, phase_column_name = 'PHASE')\n",
+    "print(phase_log)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "5a3e56d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add the generated string to the meta data of the table\n",
+    "table.meta['PH_LOG'] = phase_log"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26592cbb",
+   "metadata": {},
+   "source": [
+    "Once this is done, we can put back the table in a new `EventList` object and in a new `Observation` object. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "866007b0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "EventList\n",
+      "---------\n",
+      "\n",
+      "  Instrument       : MAGIC stereo\n",
+      "  Telescope        : MAGIC\n",
+      "  Obs. ID          : 5029747\n",
+      "\n",
+      "  Number of events : 11189\n",
+      "  Event rate       : 9.417 1 / s\n",
+      "\n",
+      "  Time start       : 56569.18112772242\n",
+      "  Time stop        : 56569.19487901596\n",
+      "\n",
+      "  Min. energy      : 1.06e-02 TeV\n",
+      "  Max. energy      : 4.71e+01 TeV\n",
+      "  Median energy    : 7.92e-02 TeV\n",
+      "\n",
+      "  Max. offset      : 2.6 deg\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# create new event list and add it to observation object\n",
+    "new_event_list = EventList(table)\n",
+    "observation._events = new_event_list\n",
```

Comment:
This is not creating a new observation. Is this safe?
I am sceptical of exposing hidden variables in a recipe

Reply:
Sure, it is not, and I think I agree with you. Maybe this can be removed since later we show how to read the new EventList and use it with the original IRFs. I think we wanted to show just how to replace the EventList in case you do not want to save the new EventList table.

Reply:
I think that this is very important to show how to save the updated event list. This line is one of the most important ones in this recipies! It should be kept, I think...
Maybe one can add some text with a warning

Reply:
@bkhelifi : The updated event list can be serialised independently, and a new observation created from that. The solution implemented here is a smart trick which experts can use, but I would be careful of exposing something which is not a part of the public API to novice users.

Reply:
as an alternative, we could show this :
```
# Create new event list and add it to observation object
new_event_list = EventList(table)
new_obs = Observation(obs_id=observation.obs_id,
                      obs_info=observation.obs_info,
                      aeff=observation.aeff,
                      edisp=observation.edisp,
                      psf=observation.psf,
                      bkg=observation.bkg,
                      rad_max=observation.rad_max,
                      gti=observation.gti,
                      events=new_event_list,
                      obs_filter=observation.obs_filter,
                     )
print(observation.events)
```